### PR TITLE
ADEN-3946 Add missing slots to AppNexus configuration

### PR DESCRIPTION
**Pull request by fraszczakszymon**

### CODE REVIEW

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js [hunk 1]
```diff
@@ -8,18 +9,25 @@ define('ext.wikia.adEngine.lookup.prebid.adapters.appnexusPlacements', [
...
-			log([
-					'getPlacement', skin, vertical,
-					'wgAdDriverAppNexusBidderPlacementsConfig variable not set or not configured correctly'
-				], 'error', logGroup
-			);
+			if (position && placementsMap[skin][vertical][position]) {
+				return placementsMap[skin][vertical][position];
+			}
+
+			// TODO: remove if statement after switching config from old structure (conf[skin][vertical] = pId)
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js:20]: INFO Complete the task associated to this TODO comment. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS1135)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 1 issue
- INFO 1 info

Watch the comments in this conversation to review them.

File: extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js (hunk 1 above)
Comment (fraszczakszymon) [reply extensions/wikia/AdEngine/js/lookup/prebid/adapters/appnexusPlacements.js:20]: Not today, Mr Sonar.

**Review by nandy-andy — APPROVED** [review]
LGTM

**Pull request merged**